pkg/console/operator/sync_v400.go

```diff
@@ -160,13 +180,28 @@ func sync_v400(co *consoleOperator, originalOperatorConfig *operatorv1.Console,
 	}
 
 	defer func() {
-		logrus.Printf("sync loop 4.0.0 complete:")
-		logrus.Printf("\t service changed: %v", svcChanged)
-		logrus.Printf("\t route changed: %v", rtChanged)
-		logrus.Printf("\t configMap changed: %v", cmChanged)
-		logrus.Printf("\t secret changed: %v", secChanged)
-		logrus.Printf("\t oauth changed: %v", oauthChanged)
-		logrus.Printf("\t deployment changed: %v", depChanged)
+		logrus.Printf("sync loop 4.0.0 complete")
```

Comment:
Reduce log spam.  There is nothing useful in just stating `true/false` a resource changed.  But if it changed, logging the version may help us know that something fluttered for some reason (and if the rest of the logs are unclear, we will know we need to add additional logging).